beacon-chain/BUILD.bazel

```
@@ -93,6 +93,12 @@ go_test(
     deps = ["@com_github_urfave_cli//:go_default_library"],
 )
 
+go_path(
+    name = "beacon-chain-path",
```

```suggestion
    name = "go_path",
```

This seems more canonical to me, imo
```
bazel build //beacon-chain:go_path
```